DESCRIPTION

```diff
@@ -19,4 +19,5 @@ Imports:
     thor
 Suggests:
     mockery,
-    testthat
```

Comment:
can these be alphabetical (though see later for a suggested alternative dependency)